librz/core/tui/panels.c

```diff
@@ -38,6 +38,11 @@
 #define PANEL_CMD_HEXDUMP       "xc"
 #define PANEL_CMD_CONSOLE       "$console"
 
+static const char *panel_command_opt[] = {
```

Comment:
We usually store such lists in `librz/core/tui/modes.c`. Since there are plans to use the common code between various visual and panels mode in the future, it would make a better sense, IMHO: https://github.com/rizinorg/rizin/issues/384

Reply:
Ok, I will fix it.